components/flags.ts

```diff
@@ -85,6 +85,14 @@ const defaultFlags: Flags = {
         desc: "Forcibly disable installed mod checks",
         default: false,
     },
+    experimentalHMR: {
+        desc: "[Experimental] Toggle hot reloading of contracts",
+        default: false,
+    },
+    developmentPluginDevHost: {
+        desc: "[Development only] Toggle loading of plugins with a .ts/.cts extension inside the /plugins folder",
```

Comment:
Worth noting here too that this requires the dev workspace, not just that it's dev only.

Reply:
Changed it to: `[Development - Workspace required] Toggle loading of plugins with a .ts/.cts extension inside the /plugins folder` and force pushed.